lib/Extractor/KLEEBuilder.cpp

```diff
@@ -188,11 +188,12 @@ ref<Expr> ExprBuilder::subnuwUB(Inst *I) {
 
 ref<Expr> ExprBuilder::mulnswUB(Inst *I) {
    const std::vector<Inst *> &Ops = I->orderedOps();
-   ref<Expr> L = SExtExpr::create(get(Ops[0]), 2*I->Width);
-   ref<Expr> R = SExtExpr::create(get(Ops[1]), 2*I->Width);
+   Expr::Width Width = get(Ops[0])->getWidth();
```

Comment:
Can you please point me to that test case? The result of a ``mul`` inst is always the same width as the operands.